packages/react-button/src/utils/useToggleable.ts

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import { useControllableState } from '@fluentui/react-utilities';
+import type { ButtonState } from '../components/Button/Button.types';
+import type { ToggleButtonProps, ToggleButtonState } from '../components/ToggleButton/ToggleButton.types';
+
+export function useToggleable<
```

Comment:
nit: toggleable sounds like a funny word to me 😅

```suggestion
export function useToggleState<
```